index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" crossorigin="anonymous">
+    <title>Adrian's Portfolio</title>
+</head>
+<body>
+    <header>
+
+        <!-- This is the mobile nav -->
+
+        <div class="nav_container">
+            <a href="#" class="logo">
+              <img src="images/MyLogo.svg" alt="Logo" />
+            </a>
+            <nav class="dropdown">
+              <img src="images/NavMenu.svg" alt="Nav Menu" class="nav_menu" />
+              <ul class="mobile_nav">
+                <!-- <li class="link"><a href="#Portfolio" class="menulink">Portfolio</a></li>
+                <li class="link"><a href="#About">About</a></li>
+                <li class="link"><a href="#Contact">Contact</a></li> -->
+              </ul>
+            </nav>
+
+            <!-- This is the desktop nav -->
+
+        <!-- <nav class="desktop_nav">
+            <ul class="desktop_ul">
+                <li class="desktop_list">
+                    <a href="#Portfolio" class="desk_info">Portfolio</a>
+                </li>
+                <li class="desktop_list">
+                    <a href="#About" class="desk_info">About</a>
+                </li>
+                <li class="desktop_list">
+                    <a href="#Contact" class="desk_info">Contact</a>
+                </li>
+            </ul>
+        </nav> -->   
```

Comment:
### OPTIONAL 

I'll mark this as a suggestion rather than a requirement so you can apply this practice on your feature repos 🧐
Do not commit old or new pieces of code as inline comments. They will make your project look messy and unorganized. If you have code for the desktop version you can always create another feature branch and get ahead of your tasks but maintaining differentiation🔥